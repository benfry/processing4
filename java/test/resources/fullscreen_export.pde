int x = 0;

void setup() {
  background(0);
  noStroke();
  fill(102);
}

void draw() {
  rect(x, height*0.2, 1, height*0.6);
  x = x + 2;
}
