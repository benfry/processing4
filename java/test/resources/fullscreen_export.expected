import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class fullscreen_export extends PApplet {

int x = 0;

 public void setup() {
  background(0);
  noStroke();
  fill(102);
}

 public void draw() {
  rect(x, height*0.2f, 1, height*0.6f);
  x = x + 2;
}


    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "--full-screen", "--bgcolor=null", "--hide-stop", "fullscreen_export" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}
